gradoop-store/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>gradoop-parent</artifactId>
+        <groupId>org.gradoop</groupId>
+        <version>0.3.3-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>gradoop-store</artifactId>
+    <packaging>pom</packaging>
+    <name>Gradoop Store</name>
+    <description>GRADOOP : Graph Store Analytics on NoSQL Layer</description>
```

Comment:
That sounds like Gradoop is a analytic platform on SQL layer. I think a better description will be : `Extended IO functionalities for NoSQL databases`